tests/test_unittest/test_meta.py

```diff
@@ -0,0 +1,15 @@
+import ast
+
+import gt4py.utils as gt_util
+
+
+class TestFullNameCreator:
```

Comment:
Looking at the length of the `get_qualified_name` method I proposed and it's simplicity we can skip this UnitTest. It's a little exessive to me, but opinions may differ.

Reply:
It's actually a good thing we had the unit test - we had the order of the recursion incorrect in building the qualified name.

Reply:
Hehe, never say something is to simple for a test they use to say.